### PR TITLE
Magento/devdocs. Issue #8897.Set docroot - remove mention of setup wizard in 2.37

### PR DESCRIPTION
**Pull request by EvheniiaKolkhiieva**
## Purpose of this pull request

This pull request (PR) fixes an issue https://github.com/magento/devdocs/issues/8897
## Affected DevDocs pages

-  https://devdocs.magento.com/cloud/before/before-setup-env-install.html#set-the-docroot

### CODE REVIEW

File: src/cloud/before/before-setup-env-install.md [hunk 1]
```diff
@@ -68,7 +68,7 @@ To create authentication keys through the Magento Marketplace:
 
 ## Set the docroot
 
-Set the `docroot` to the `/magento` directory until you complete all setup. If you change the `docroot` to `/magento/pub` prior to completion, you will encounter issues running the Web Setup Wizard.
+Set the `docroot` to the `/magento` directory until you complete all setup.
```
Comment (BarnyShergold) [inline src/cloud/before/before-setup-env-install.md:71]:
```suggestion
Set the `docroot` to the `/magento` directory until you complete the setup.
```
First sentence doesn't scan 😄

**Review by BarnyShergold — CHANGES_REQUESTED** [review]

**Review by hguthrie — APPROVED** [review]

**Review by BarnyShergold — APPROVED** [review]
Thanks

**Review by bdenham — APPROVED** [review]

**Comment by hguthrie** [comment]
@magento import code to magento-devdocs/devdocs

**Comment by magento-engcom-team** [comment]
@hguthrie the branch with code successfully imported into`magento-devdocs/devdocs` repository. Branch name: `imported-magento-devdocs-8905`.

**Pull request merged**